README.md

```diff
@@ -9,10 +9,16 @@ We are working not only in a protocol, but also the firmware for affordable devi
 
 * The main software running on Turpial devices (ESP32) can be found [here](https://github.com/btcven/locha-mesh-app)
 
+* Our firmware for the ESP32 (Turpial) chip boards can be found [here](https://github.com/btcven/turpial-firmware)
+
+* Our firmware for the CC1312R radio modules can be found [here](https://github.com/btcven/radio-firmware)
```

Comment:
Although the Turpial has an CC1312 TI radio module, in practice the firmware can be installed on any platform capable of running contiki-ng:  https://github.com/contiki-ng/contiki-ng/wiki#the-contiki-ng-platforms with special attention to: https://github.com/contiki-ng/contiki-ng/wiki/Platform-simplelink 

So i think _"Our firmware for the CC1312R radio modules ..."_ should be changed to a more generic name or simply _"Our firmware for the radio modules ..."_

Reply:
Thanks for contributing @xmrhaelan :octocat:

Reply:
Thank you! Made a few more changes based on this feedback and information. Also edited the introduction paragraph to hopefully help demonstrate wider use-case and show what I perceive as the good motivations behind this project :)